packages/test/test-end-to-end-tests/src/test/blobs.spec.ts

```diff
@@ -241,6 +241,16 @@ describeNoCompat("blobs", (getTestObjectProvider) => {
 
 		const attachOpP = new Promise<void>((resolve, reject) =>
 			container1.on("op", (op) => {
+				if (op.contents?.type === "groupedBatch") {
```

Comment:
Shouldn't we not need this code? I believe you addressed the issue, and presence of this code suggests that service will not get info it needs.